app/models/hackbot/dispatcher.rb

```diff
@@ -2,8 +2,7 @@ module Hackbot
   class Dispatcher
     CONVERSATION_TYPES = [
       Hackbot::Conversations::CheckIn,
-      Hackbot::Conversations::Help,
-      Hackbot::Conversations::Mention
+      Hackbot::Conversations::Help
```

Comment:
Yeah, not sure what I was going for previously. I think I meant to say that it's okay to have PRs that are for "improving Hackbot" and include a variety of improvements.